templates/tuf-vault.tmpl.yaml

```diff
@@ -42,10 +42,11 @@ spec:
             cpu: {{ .tuf_vault_cpu }}
             memory: {{ .tuf_vault_mem }}
         livenessProbe:
-          initialDelaySeconds: 300
-          periodSeconds: 30
-          tcpSocket:
+          initialDelaySeconds: 20
+          periodSeconds: 5
+          httpGet:
             port: 8200
+            path: /v1/sys/init
```

Comment:
not sure about this, does init endpoint respond with 200 when the container is uninitialized?

Reply:
Yes, it responds with 
```
HTTP/1.1 200 OK
Cache-Control: no-store
Connection: keep-alive
Content-Length: 22
Content-Type: application/json
Date: Thu, 11 Jan 2018 13:38:45 GMT
Server: nginx/1.13.6
Vary: Accept-Encoding

{
    "initialized": false
}
```